voxblox/include/voxblox/core/layer_inl.h

```diff
@@ -210,16 +210,32 @@ bool Layer<VoxelType>::addBlockFromProto(const BlockProto& block_proto,
 template <typename VoxelType>
 bool Layer<VoxelType>::isCompatible(const LayerProto& layer_proto) const {
   bool compatible = true;
-  compatible &= (layer_proto.voxel_size() == voxel_size_);
+  compatible &= (std::fabs(layer_proto.voxel_size() - voxel_size_) <
+                 std::numeric_limits<FloatingPoint>::epsilon());
   compatible &= (layer_proto.voxels_per_side() == voxels_per_side_);
   compatible &= (getType().compare(layer_proto.type()) == 0);
+
+  if (!compatible) {
+    LOG(INFO) << "Voxel size of the loaded map is: " << layer_proto.voxel_size()
```

Comment:
I would make this a LOG(ERROR) or LOG(WARNING) at the very least. This is not sth that should happen often and LOG(INFO) is easily ignored :)

Reply:
Great, and I'll add a check for double vs float maps into this message as well (since it's trivial).